static/css/biomedical/biomedical_landing.scss

```diff
@@ -0,0 +1,265 @@
+    --title-caption-line-height: 24px;
+    --title-font-size: 28px;
+    --title-line-height: 36px;
+  }
+}
+
+/**
+ * Override base styles
+ */
+
+#main-footer {
+  background-color: white;
+  //padding: 0px;
+}
+
+.main-content {
+  // start content at top, but leave room for box shadow on #main-header
+  margin: 3px;
+}
+
+/** 
+ * Landing page specific styles
+ */
+
+.background-highlight {
+  background: $section-background-color;
+}
+
+.blurb-container {
+  color: $main-text-color;
+  display: flex;
+  flex-direction: column;
+  font-size: var(--blurb-text-font-size);
+  font-weight: var(--blurb-text-font-weight);
+  line-height: var(--blurb-text-line-height);
+  padding: 64px 15px;
+
+  a {
+    color: $highlight-text-color;
+  }
+
+  h2 {
+    color: $header-text-color;
+    font-size: var(--blurb-header-font-size);
+    font-weight: var(--blurb-header-font-weight);
+    line-height: var(--blurb-header-line-height);
+    margin-bottom: 22px;
+  }
+
+  .blurb-text {
+    margin-bottom: 22px;
+
+    a {
+      text-decoration-line: underline;
+    }
+  }
+}
+
+a.explore-structure-link {
+  align-items: center;
+  display: flex;
+  gap: 8px;
+}
+
+.numbers-callout-container {
+  @extend .blurb-container;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 32px;
+  justify-content: space-evenly;
+  text-align: center;
+
+  .number-callout {
+    display: flex;
+    flex-direction: column;
+    gap: var(--number-callout-gap);
+    font-size: var(--number-callout-label-font-size);
+    font-weight: var(--number-callout-label-font-weight);
+    line-height: var(--number-callout-label-line-height);
+  }
+
+  .number-highlight {
+    color: $highlight-text-color;
+    font-size: var(--number-callout-highlight-font-size);
+    font-weight: var(--number-callout-highlight-font-weight);
+    line-height: var(--number-callout-highlight-line-height);
+  }
+}
+
+#sample-questions h2 {
+  padding-bottom: 22px;
+}
+
+.sample-analyses-container {
+  @extend .blurb-container;
+  margin-top: 24px;
+}
+
+.title-content-container {
+  @extend .blurb-container;
+  border-bottom: 1px solid #dde3ea;
+  box-sizing: border-box;
+  flex-direction: row;
+  gap: 64px;
+  padding-bottom: 0px;
+
+  .title-content {
+    flex-grow: 1;
+
+    h1 {
+      color: $header-text-color;
+      font-size: var(--title-font-size);
+      font-weight: var(--title-font-weight);
+      line-height: var(--title-line-height);
+      margin-bottom: 8px;
+    }
+
+    h2 {
+      color: $header-text-color;
+      font-size: var(--subtitle-font-size);
+      font-weight: var(--subtitle-font-weight);
+      line-height: var(--subtitle-line-height);
+      margin-bottom: 8px;
+    }
+
+    h3 {
+      color: $header-text-color-light;
+      font-size: var(--title-caption-font-size);
+      font-weight: var(--title-caption-font-weight);
+      line-height: var(--title-caption-line-height);
+    }
+  }
+
+  .thematic-image {
+    margin-top: auto;
+  }
+}
+
+.card {
```

Comment:
curious what card this is referring to because I see that there is a file for Card with its own css

Reply:
It was originally there to allow for repsonsive text resizing, with the css card file as its own css. I've switched to doing all rendering with a single react component, so all styling is now in the Card file.